draft-ietf-jsonpath-base.md

```diff
@@ -1420,7 +1420,8 @@ This section defines the extension point as well as four function
 extensions that use this extension point.
 While these mechanisms are designed to use the extension point,
 they are an integral part of the JSONPath specification and are
-mandatory to implement.
+expected to be implemented like any other integral part of this
+specification.
```

Comment:
The change is a bit nuanced from my perspective. What was the rewording trying to get across? It reads about the same to me.

Reply:
Avoiding the big word "mandatory".
(There is a long discussion about compliance vs. interoperability in the IETF.)

Reply:
Ok, thanks. Maybe I don't need to know about that discussion, but if you have a link handy, that would be good.

Reply:
If you are into internet history, you want to be on the Internet History list: https://elists.isoc.org/mailman/listinfo/internet-history

There are probably dozens, but here is one recent thread that mentioned how interoperability won over compliance:
https://elists.isoc.org/pipermail/internet-history/2022-March/007880.html